src/cli/flags/restore_config.go

```diff
@@ -0,0 +1,30 @@
+package flags
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/alcionai/corso/src/pkg/control"
+)
+
+const (
+	CollisionsFN  = "collisions"
+	DestinationFN = "destination"
+)
+
+var (
+	CollisionsFV  string
+	DestinationFV string
+)
+
+// AddRestoreConfigFlags adds the restore config flag set.
+func AddRestoreConfigFlags(cmd *cobra.Command) {
+	fs := cmd.Flags()
+	fs.StringVar(
+		&CollisionsFV, CollisionsFN, string(control.Skip),
+		"How to handle item collisions: "+string(control.Skip)+", "+string(control.Copy)+", or "+string(control.Replace))
+	cobra.CheckErr(fs.MarkHidden(CollisionsFN))
+	fs.StringVar(
+		&DestinationFV, DestinationFN, "",
+		"Overrides the destination where items get restored.  '/' places items back in their original location.")
```

Comment:
yes, no.  This will need longer explanation somewhere, but I'm not sure where to put it, and I don't think this is the right place, per se.  Will be working with Georgi to figure that out.